cmd/end2endtest/helpers.go

```diff
@@ -420,13 +545,22 @@ func censusParticipantKey(voterAccount *ethereum.SignKeys, censusType string) ([
 	return key, nil
 }
 
-// matchResult compare the expected vote results base in the overwrite applied, with the actual result returned by the API
-func matchResult(results [][]*types.BigInt, expectedResult [][]string) bool {
-	// only has 1 question
-	for q := range results[0] {
-		if !(expectedResult[0][q] == results[0][q].String()) {
-			return false
+func matchResults(results [][]*types.BigInt, expectedResults [][]*types.BigInt) bool {
+	// iterate over each question to check if the results match with the expected results
+	for i := 0; i < len(results); i++ {
+		for q := range results[i] {
+			if !(expectedResults[i][q].String() == results[i][q].String()) {
+				return false
+			}
 		}
 	}
 	return true
 }
+
+func votesToBigInt(votes []uint64) []*types.BigInt {
```

Comment:
it would be much prettier for the caller if this would be 
`votesToBigInt(votes ...int)`

so the calls would be

`resultsCh1 = votesToBigInt(0, 10, 20, 0, 0, 0, 10)`